turbo.json

```diff
@@ -35,7 +35,8 @@
     },
     "test": {
       "dependsOn": ["^test"],
-      "cache": false
+      "cache": false,
+      "passThroughEnv": ["BUCKET_ENV_MINIO"]
```

Comment:
not sure if this was the right way to do it ? , but env vars are getting passed now and test suite is running as expected.